In [37]:
import pandas as pd
import numpy as np

data = pd.read_csv('/workspaces/Coding/IIMR Term 1/Group Assignment/q30 data.csv')
df = pd.DataFrame(data)

In [35]:
# Calculate Sample Mean and Standard Deviation
x̄ = np.mean(df['hrs'])
s = np.std(df['hrs'])
n = len(df['hrs'])
μ = 6.4
dframe = μ - 1

print('x̄: ', x̄)
print('s: ', s)
print('n: ', n)
print('μ: ', μ)
print('df: ', dframe)

x̄:  7.0
s:  2.3970815588961507
n:  40
μ:  6.4
df:  5.4


**Step 1: Hypothesis**<br>
Ho: μ = 6.4 <br>
H1: μ ≠ 6.4

**Step 2: T Statistic**

In [36]:
print('T Statistics: (x̄ - μ)/ (s / √ n)')

t = (x̄ - μ) / (s / np.sqrt(n))

print('=', t)

T Statistics: (x̄ - μ)/ (s / √ n)
= 1.5830638628539269


**Step 3: P-Value Approach**<br>
Alpha = 0.05<br>
At df = 39 & t = 1.58<br>
p-value: (0.1 - 0.2)<br><br>
P-value (0.1 - 0.2) > Alpha (0.05)<br>
Therefore, H0 cannot be rejected.

**Step 4: Critical Value Approach**<br>
T_cutoff = +- 2.023<br>
Reject H0 if t <= -2.023 OR<br>
&emsp; &emsp; &emsp; &emsp; &nbsp; if t >= 2.023<br>
Therefore, H0 canot be rejected.

**Conclusion**<br>
Population mean number of hours married men are spending in child care does not differ from the mean reported by <i> Time </i> in any area.